README.md

```diff
@@ -154,6 +154,14 @@ then 4 times in a row and issue an appropriate alert.
 
 ## Running a test environment
 
+### Kwok Docker Container
+
+This repository contains a Dockerfile to build a container for the Kwok Kubernetes tool. Navigate to the 'kwok' directory, build the Docker container for the Kwok Kubernetes tool using the provided Dockerfile, and then run the created Docker image interactively.
+
+1. `cd kwok`
+2. `docker build -t kwok-image .`
+3. `docker run -it --rm kwok-image:latest`
+
```

Comment:
Missing motes about how to use the image as a cluster

Reply:
Overall, I feel this PR should have been about how to USE the KWOK image to function as a pseudo-cluster rather than BUILDING KWOK - as KWOK is already being offered as a binary and as a pre-built image.

Reply:
Done

Reply:
We need a way to connect to the cluster in the container from outside the container. The given `exec` command only does that from inside the container.

Also, can you please use podman instead of docker?

Reply:
Updated

Reply:
The image doesn't have bash or sh. another way to run outside the container is to add the Kube config and export it.

Reply:
or we may have to revert to use ubi image. @ifireball  wdyt?

Reply:
what does running outside of the container have to do with having a shell inside the container?

we need a way to setup a predictable port and predictable credentials for the cluster in the container so that we know how to connect to it from kubectl/oc running outside.

Reply:
In the updated step we can access using the cluster name and port is also exposed.

Reply:
Does `kwokctl get clusters` print how to connect to the cluster? 

Consider that what we are going to have in the end is a test in Go that is going to bring up the kwok image, run our {{splunk-to-segment.sh}} script while providing a kube config and then inspect the output. Its preferable of we don't have to run any special kwok commands in order to figure out how to connect to it.

Reply:
@ifireball it does but this should also be printed [when the container starts](https://github.com/kubernetes-sigs/kwok/blob/90f923e7c8320d79e98b5cf3dbd0edfce35a1fa7/images/cluster/entrypoint.sh#L27-L39). It's mostly static so we can probably commit this kubeconfig. The port is configurable and defaults to [8080](https://github.com/kubernetes-sigs/kwok/blob/90f923e7c8320d79e98b5cf3dbd0edfce35a1fa7/images/cluster/Dockerfile#L21).

@srivickynesh we should show a sample `kubectl` command using such a kubeconfig.

Reply:
@amisstea ok.

@srivickynesh can we get this done by end of sprint?